### DISCUSSION TITLE
COD WAW graphics Outdated please delete

### DISCUSSION THREAD

**Discussion by McNuttGaming2** [nzp-team/nzportable#440] [Other Content Releases]
![graphics](https://github.com/nzp-team/nzportable/assets/150686160/b861654f-3afd-4668-ac3f-e3d06ede227c)
Not finished
Includes hud and main menu
[Download (PC, Switch)](https://www.mediafire.com/file/ehvnrbzg9045xpp/cod_waw_graphics_for_nzp.zip/file)
[Download (3DS, PSP, Vita)](https://www.mediafire.com/file/u4j8wgr11qct7bm/cod_waw_graphics_for_nzp_psp.zip/file)

**Comment by Ellis243454** [comment]
Will this work for ds

**Reply by McNuttGaming2** [reply]
Yes but you have to use the psp one
